smoke-tests/src/test/java/com/splunk/opentelemetry/SpringBootSmokeTest.java

```diff
@@ -16,43 +16,47 @@
 
 package com.splunk.opentelemetry;
 
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
 import io.opentelemetry.proto.collector.trace.v1.ExportTraceServiceRequest;
 import java.io.IOException;
 import java.util.Collection;
 import java.util.Collections;
 import java.util.Map;
-import java.util.jar.Attributes;
-import java.util.jar.JarFile;
+import java.util.stream.Stream;
 import okhttp3.Request;
 import okhttp3.Response;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.ValueSource;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 
 class SpringBootSmokeTest extends SmokeTest {
 
-  protected String getTargetImage(int jdk) {
-    return "open-telemetry-docker-dev.bintray.io/java/smoke-springboot-jdk" + jdk + ":latest";
+  private static Stream<Arguments> springBootConfigurations() {
+    return Stream.of(
+        arguments(
+            new SpringBootConfiguration(8),
+            new SpringBootConfiguration(11),
+            new SpringBootConfiguration(14),
```

Comment:
No need for 14. Rule of thumb is "all LTS plus the latest if not LTS"

Reply:
I just converted what there was already: https://github.com/signalfx/splunk-otel-java/blob/main/smoke-tests/src/test/java/com/splunk/opentelemetry/SpringBootSmokeTest.java#L44

But will remove.